src/App.tsx

```diff
@@ -1,167 +1,21 @@
-import { Loader } from './components/Loader';
-
 import './App.scss';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import Laout from './components/Laout';
```

Comment:
There seems to be a typo in the import statement for the `Laout` component. It is likely intended to be `Layout`. Please verify the correct component name and update the import statement accordingly.